admin/spec/features/products_spec.rb

```diff
@@ -12,4 +12,48 @@
     expect(page).to have_content("$19.99")
     expect(page).to be_axe_clean
   end
+
+  it "can delete multiple products at once", js: true do
```

Comment:
Could we add the a11y expectation at the end of the new examples? :pray:

Reply:
Added and fixed an issue with flash messages in https://github.com/solidusio/solidus/pull/5259